stdlib/public/SDK/XCTest/XCTest.swift

```diff
@@ -55,11 +55,11 @@ public extension XCUIElement {
 func _XCTRegisterFailure(_ expected: Bool, _ condition: String, _ message: @autoclosure () -> String, _ file: StaticString, _ line: UInt) {
   // Call the real _XCTFailureHandler.
   let test = _XCTCurrentTestCase()
-  _XCTPreformattedFailureHandler(test, expected, file.description, line, condition, message())
+  _XCTPreformattedFailureHandler(test, expected, file.description, Int(line), condition, message())
```

Comment:
Why not just change the argument to Int here too?

Reply:
Because that just meant a cast in more places. This is called from APIs such as `XCTFail`, and we can't change the signature there. So doing one cast here was less change